tests/foreman/ui/test_errata.py

```diff
@@ -467,6 +496,35 @@ def test_positive_content_host_library(session, module_org, vm):
         assert content_host_erratum[0]['Id'] == CUSTOM_REPO_ERRATA_ID
 
 
+@tier3
+def test_positive_content_host_search_type(session, erratatype_vm):
+    """ Search for errata on a content_host by type
+
+    :id: 59e5d6e5-2537-4387-a7d3-637cc4b52d0e
+
+    :Setup: Content Host with applicable security errata
+
+    :Steps: Search for errata on content host by type (ie 'type = security')
+
+    :BZ: 1653293
+
+    :CaseLevel: Integration
+    """
+
+    pkgs = " ".join(FAKE_9_YUM_OUTDATED_PACKAGES)
+    assert _install_client_package(erratatype_vm, pkgs, errata_applicability=True)
+
+    with session:
+        ch_erratum = session.contenthost.search_errata(
+            erratatype_vm.hostname, "type = security", environment='Library Synced Content')
```

Comment:
I can't really make it out of upstream PR, but I assume there are various attributes that can be used for searching, and "type = security" is just an example. Is that correct? Would it make sense to extend this test to try some other searches here as well?

Reply:
There are other tests that do searches. I was thinking of checking other types and other search in this tests, but we didn't seem to have any existing repos that had enough of a mix of errata for that. I am sure we can add some in the future, but I wanted to get this PR out fast since it was a closed loop one. 

Also, I would want to balance home much time it would take to run this test.